mysqldef/time_test.go

```diff
@@ -289,6 +289,26 @@ func (s *testTimeSuite) TestYear(c *C) {
 
 }
 
+func (s *testTimeSuite) getLocation(c *C) *time.Location {
+	locations := []string{"Asia/Shanghai", "Europe/Berlin"}
+	timeFormat := "Jan 2, 2006 at 3:04pm (MST)"
```

Comment:
timeFormat and value variable is from golang.org/time ParseInLocation example.
i think here it is no need to use TimeFormat, for it is type of  `2006-01-02 15:04:05`, i will have to modify value variable.